src/common/components/intro/index.tsx

```diff
@@ -19,7 +19,17 @@ interface Props {
 export default (props: Props) => {
     const hideIntro = () => props.hideIntro();
 
-    if (!props.global.intro) {
+    const [isLoggedIn, setIsLoggedIn] = useState(false)
+
+    useEffect(() => {
+        const verification = localStorage.getItem("ecency_active_user")
```

Comment:
I don't think you would need localStorage, you can use activeUser with props, no? Check `entry-index.tsx`

Reply:
Ok will fix